kani-driver/src/cbmc_output_parser.rs

```diff
@@ -35,6 +35,8 @@ use std::os::unix::process::ExitStatusExt;
 use std::path::PathBuf;
 use std::process::{Child, ChildStdout};
 
+const RESULT_ITEM_PREFIX: &str = "  {\n    \"result\":";
```

Comment:
Should we use regex instead or will this run all the time? I'm worried this is very fragile.

Reply:
There were a few things I considered here, for example trimming the string without delimiters (so we can check against `"{\"result\":"`). The problem is that trimming that string could impact performance: the one that caused the crash in #2169 is 921K lines (33MB). So I went for the most performant check (this one), which should run all the time.